src/lcm/LifeCycleStates.cc

```diff
@@ -1761,6 +1891,8 @@ void LifeCycleManager::trigger_saveas_failure(int vid)
 
         if ( auto image = ipool->get(image_id) )
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This snippet may contain an accidental use of incorrect operation. Make sure that the operation being performed is in fact what you intended.